.github/workflows/split_composer_test.yaml

```diff
@@ -47,7 +47,7 @@ jobs:
 
             -   uses: shivammathur/setup-php@v2
                 with:
-                    php-version: 7.2
+                    php-version: 7.3
```

Comment:
:+1:

Reply:
Yup, Needed 7.2 for the older version of code sniffer used before this PR.
But that was updated in this PR, so I can also bump this!